securedrop/manage.py

```diff
@@ -21,20 +21,23 @@
 
 sys.path.insert(0, "/var/www/securedrop")  # noqa: E402
 
-import qrcode
-from sqlalchemy.orm.exc import NoResultFound
+import qrcode  # noqa: E402
+from sqlalchemy.orm.exc import NoResultFound  # noqa: E402
 
-os.environ['SECUREDROP_ENV'] = 'dev'  # noqa
 
-from db import db
-from models import (
+if not os.environ.get("SECUREDROP_ENV"):
+    os.environ['SECUREDROP_ENV'] = 'dev'  # noqa
```

Comment:
@zenmonkeykstop This fixes the alembic tests. Thanks for explaining what the problem was.